client/src/ECP.ts

```diff
@@ -181,13 +183,14 @@ export class ECP {
 
 	// Helper for sending a /input request to the device that can be handled via roInput
 	public async sendInput({
-		params = {}
+		params = {},
+		options = {} as HttpRequestOptions
 	} = {}) {
-		await this.device.sendEcpPost(`input`, params);
+		await this.device.sendEcpPost(`input`, params, undefined, options);
 	}
 
-	public async getActiveApp() {
-		const result = await this.device.sendEcpGet(`query/active-app`);
+	public async getActiveApp(options: HttpRequestOptions = {}) {
+		const result = await this.device.sendEcpGet(`query/active-app`, undefined, options);
```

Comment:
I wish we didn't have to pass `undefined` for param3. :(

Reply:
We could switch the second param to have something be something like 
```
{
  params: params
  options: options
}
```
but that would be a breaking change for anyone using sendEcpGet

Reply:
Yeah. I had discussed how to fix that in another comment, but you answered that as well. All good.